aitinero/src/app/home/duration/page.tsx

```diff
@@ -0,0 +1,33 @@
+"use client";
+
+import { Input } from "@/components/ui/input";
+import { Button } from "@/components/ui/button";
+import FormWrapper from "@/components/FormWrapper";
+import useAppFormContext from "@/lib/hooks/useAppFormContext";
+import  {useRouter} from "next/navigation";
+
+
+export default function DateForm() {
+    const router = useRouter();
+    const {register, trigger, formState} = useAppFormContext();
+    const {isValid} = formState;
+
+    return (
+        <FormWrapper>
+            <div className="flex flex-col place-items-center">
+                <div className="text-slate-700 text-xl font-semibold">
+                <p><strong>When</strong> do you want to go?</p>
+                </div>
+                <div className="text-white my-2 flex flex-row">
+                    <Input type="date" className="rounded-md p-2 w-24 bg-slate-800 mx-2" {...register('startDate')}/>
+                    <Input type="date" className="rounded-md p-2 w-24 bg-slate-800 mx-2" {...register('endDate')}/>
```

Comment:
Can increase the width of these two inputs? Quite tight now.

Reply:
<img width="838" alt="image" src="https://github.com/LZher385/AItinero/assets/71116618/32f4f2fb-2418-4eaf-b881-2838c3ded211">